domains/anubhav823.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "anubhav823' personal developer website",
+  "repo": "",
+  "owner": {
+    "username": "anubhav823",
+    "email": "anubhav823@gmail.com"
+  },
+  "record": {
+    "CNAME": "anubhav823.github.io/wordle"
```

Comment:
You cannot use paths in CNAME records, if you'd like to redirect to `anubhav823.github.io/wordle` then please use a URL record.

Reply:
I have changed the record type.